api/src/main/java/org/openmrs/module/pihcore/config/ConfigLoader.java

```diff
@@ -42,7 +43,7 @@ public static String getPihConfigurationDirRuntimeProperty(String defaultValue)
      * Loads Configuration based on configuration in the runtime properties file
      */
     public static ConfigDescriptor loadFromRuntimeProperties() {
-        return load(getRuntimeConfiguration("pihcore"));
+        return load(getRuntimeConfiguration("default,site-default"));
```

Comment:
By default, if no "pih.config" runtime-property is found, OpenMRS will configure using the following two config files:

configuration/pih/pih-config-default.json
configuration/pih/pin-config-site-default.json

This allows someone to fire up a base configuration without the hacky step to stop and update their runtime.properties file.

I'm setting up these files via sym links in the various project directories.